src/components/ReleaseCard/index.jsx

```diff
@@ -300,7 +300,7 @@ function ReleaseCard(props) {
       </CardContent>
       <CardActions className={classes.cardActions}>
         <Link className={classes.link} to={`/releases/${release.name}`}>
-          <Button color="secondary">
+          <Button color="secondary" disabled={!user}>
```

Comment:
I think this button should stay enabled even if the user is not logged in. However, it should always read "View" in that case.

Reply:
Ah, makes sense. I think we can keep the current behavior of showing "Update" and have the actions in that view disabled. It will tell the user that updating a release is possible.